tests/mpi/tria_copy_triangulation.cc

```diff
@@ -109,11 +112,6 @@ void test()
       else
         Assert (false, ExcInternalError());
     };
-
-//   assert_tria_equal("tria_copy_triangulation", tr, new_tr);
-
-  Assert (tr.n_active_cells() == new_tr.n_active_cells(), ExcInternalError());
-  Assert (tr.n_levels() == new_tr.n_levels(), ExcInternalError());
```

Comment:
It's just moved up. In `cell1 != tr.end(), cell2 != new_tr.end();` the first statement was never used. However, it should be sufficient to only have one of these two if we first check that the number of active cells is the same.
Let me extract this in another PR.